README.md

```diff
@@ -2,23 +2,37 @@
 
 This is a Go command line program that will allow listing and interacting with a Pairtree without knowing anything about the Pairtree’s internal structure. 
 
+## Installation
+
+First, ensure that you have Go Version 1.22 on you system. Clone this repository
```

Comment:
I wonder if there is a way to supply this version number automatically? Javascript that pulls down the version from DockerHub's latest container? Or maybe just saying "the latest" version? I've had READMEs before with these sorts of version numbers and they become a pain to manually keep up with (e.g., I think Go is on 1.23 now?) I don't think this is a blocker for this PR, but something we might want to discuss/think about.